index.bs

```diff
@@ -568,153 +564,187 @@ XRFrame {#xrframe-interface}
+  - Passing a {{XRReferenceSpaceOptions/type}} of <dfn enum-value for="XRReferenceSpaceType">unbounded</dfn> creates an {{XRUnboundedReferenceSpace}} instance if supported by the [=XR device=].
 
-If the device is capable of reporting accurate translation, it MUST be discarded when using this [=frame of reference type=] in favor of an orientation-based estimate. If the translation reported by the device is already based on head modeling the device-provided translation MAY be preserved.
+The <dfn attribute for="XRReferenceSpace">onreset</dfn> attribute is an [=Event handler IDL attribute=] for the {{reset}} event type.
 
-NOTE: A <code>"head-model"</code> [=frame of reference type=] is useful when displaying content that cannot be viewed from arbitrary positions, such as panoramic images or videos. 
+<div class="algorithm" data-algorithm="create-frame-of-reference">
 
-<b>stage</b>
+When an {{XRReferenceSpace}} is requested, the user agent MUST <dfn>create a reference space</dfn> by running the following steps:
 
-An {{XRFrameOfReference}} with a [=frame of reference type=] of <dfn enum-value for="XRFrameOfReferenceType">"stage"</dfn> describes a space known as a <dfn for="XRStageBounds">stage</dfn>. A [=stage=] is a bounded, floor-relative play space that the user can be expected to safely be able to move within. Other XR platforms sometimes refer to this concept as "room scale" or "standing space".
+  1. Initialize [=XRReferenceSpace/session=] be the {{XRSession}} object that requested creation of a reference space.
+  1. Let |options| be the {{XRReferenceSpaceOptions}} passed to {{requestReferenceSpace()}}.
+  1. Let |type| be set to |options| {{XRReferenceSpaceOptions/type}}.
+  1. Let |referenceSpace| be set to <code>null</code>.
+  1. If |type| is {{stationary}}, let |referenceSpace| be a new {{XRStationaryReferenceSpace}}.
+  1. Else if |type| is {{bounded}}, let |referenceSpace| be a new {{XRBoundedReferenceSpace}}.
+  1. Else if |type| is {{unbounded}}, let |referenceSpace| be a new {{XRUnboundedReferenceSpace}}.
+  1. Return |referenceSpace|.
 
-Note: A [=stage=] is not intended to describe multi-room spaces, areas with uneven floor levels, or very large open areas. Future iterations of this specification may add more detailed support for tracking in those scenarios.
+</div>
+
+Issue: Describe circumstances when a reference space type might be rejected.
```

Comment:
We might want to explain the reset event a bit more here?

Reply:
This is a weird pattern that appears in most web standards I've seen when the events themselves are described not at the handler attribute but as a separate events section (which in this doc is further down). Not my favorite layout, but I'd prefer not to break precedence. 🤷‍♀️

Reply:
Fair enough.